.dotnet/src/Custom/Assistants/AssistantClient.cs

```diff
@@ -310,8 +304,9 @@ public virtual ClientResult<AssistantDeletionResult> DeleteAssistant(string assi
     /// <returns> A new thread. </returns>
     public virtual async Task<ClientResult<AssistantThread>> CreateThreadAsync(ThreadCreationOptions options = null, CancellationToken cancellationToken = default)
     {
-        ClientResult protocolResult = await CreateThreadAsync(options?.ToBinaryContent(), cancellationToken.ToRequestOptions()).ConfigureAwait(false);
-        return CreateResultFromProtocol(protocolResult, AssistantThread.FromResponse);
+        var opt = options == null ? null : options;
```

Comment:
Is this needed? Can we directly pass options to the `CreateThreadAsync` method?

Reply:
fixed.